cmd/generate/config/rules/rule.go

```diff
@@ -19,7 +19,7 @@ const (
 	identifierSuffix = `)(?:[0-9a-z\-_\t .]{0,20})(?:[\s|']|[\s|"]){0,3}`
 
 	// commonly used assignment operators or function call
-	operator = `(?:=|>|:=|\|\|:|<=|=>|:)`
+	operator = `(?:=|>|:{1,3}=|\|\|:|<=|=>|:|\?=)`
```

Comment:
This could also be `(?:=|>|:+=|\|\|:|<=|=>|:|\?=) for a bit more flexibility with the colons. Let me know if you prefer this to {1,3}.

Reply:
```
operator = `(?:=|>|:{1,3}=|\|\|:|<=|=>|:|\?=)`
```
 is good with me